docs/user/openstack/README.md

```diff
@@ -69,6 +69,27 @@ In addition, it covers the installation with the default CNI (OVNKubernetes), as
 
 ## OpenStack Requirements
 
+The OpenShift installation on OpenStack platform relies on a number of core
+services being available:
+
+- Keystone
+- Neutron
+- Nova
+  - with Metadata service enabled
+  - Placement, as it is required by Nova
+- Glance
+- Storage solution for the image registry, one of:
+  - Swift
+  - Cinder
+  - Manila
```

Comment:
Manila is _not_ a requirement, or option for the image registry at install time. I'd remove it from this list.